docs/4.2/quickstart.md

```diff
@@ -22,10 +22,10 @@ VM.
   the best place to start!
 
 This guide is only meant to demonstrate how to run Teleport in a sandbox or demo
-environment, and showcase a few basic tasks you can do with Teleport.
+environment, and showcase a few basic tasks you can do with Teleport. For additional configuration examples, see [teleport/examples](https://github.com/gravitational/teleport/tree/master/examples), within the Teleport Github repository.
 
-**You should not follow this guide if you want to set up Teleport in production.
-Instead follow the [Admin Guide](admin-guide.md)**
+!!! note "IMPORTANT"
+    You should not follow this guide if you want to set up Teleport in production. Instead follow the [Admin Guide](admin-guide.md). 
```

Comment:
Let's point to https://gravitational.com/teleport/docs/production/ instead.

Reply:
Good idea, will do!